source/hardware/controllers/denon_mc7000.rst

```diff
@@ -219,6 +223,9 @@ Deck Section
 .. hint::
    The jog sensitivity and scratch parameters (28), the default vinyl mode (30), the pitch fader ranges (32) as well as the needle drop activity (54) can be set by modifying the variables inside the :ref:`JavaScript file<denon_mc7000_uservariables>` accordingly.
 
+.. note::
+   Use :hwlabel:`SHIFT` + :hwlabel:`CENSOR` or :hwlabel:`SHIFT` + jog wheel side for a backspin while :hwlabel:`SLIP` is active. Backspin using the platter is only possible while :hwlabel:`SLIP` is disabled.
```

Comment:
press Shift + Censor _OR_ press Shift + turn wheel?
If so this needs to be clarified IMO

Reply:
Shift + jog wheel is a side effect from seeking through a track. Maybe it's better I just delete the comment and only mention the dedicated backspin function on the censor button. Gonna send an update later today. Cheers!